pkg/controllers/utils/helpers.go

```diff
@@ -20,6 +20,16 @@ import (
 	"antrea.io/nephe/apis/crd/v1alpha1"
 )
 
+const (
+	VpcIndexerByAccountNameSpacedName = "namespace.accountname"
```

Comment:
You cannot put this in utils.. utils are like helper, and this is used by module. Find a better place or create a new file. @archanapholla please help Bangqi locate one

Reply:
Since inventory.go and store/vpc_inventory.go are both in controllers/inventory now, there are three possible ways to solve this:
1. Still put it in vpc_inventory.go
2. Create a new folder with a new file to place these const, it can be imported by both of these const
3. put it out of controllers/inventory

Discussed this with @archanapholla , for now I chose the second, in the new common/common.go, but we can discuss it later tonight